azure-pipelines.yml

```diff
@@ -0,0 +1,65 @@
+trigger:
+  branches:
+    include:
+      - main
+pr:
+  branches:
+    include:
+      - main
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+
+- script: echo $PATH
```

Comment:
Why are we doing this? Just curious.

Reply:
My guess it to debug what is in the PATH variable. I think it should be removed now.